service/main.go

```diff
@@ -199,14 +199,15 @@ func (s *service) indexHeights(heightsToIndex []int) error {
 	semaphoreLimiter = semaphore.NewWeighted(s.concurrency)
 
 	for _, height := range heightsToIndex {
-		indexingProcesses.Add(4)
+		indexingProcesses.Add(5)
 
-		err := semaphoreLimiter.Acquire(context.Background(), 4)
+		err := semaphoreLimiter.Acquire(context.Background(), 5)
```

Comment:
can this value be changed to a constant? So is easier to reason on the meaning of the number